src/styles/card.css

```diff
@@ -0,0 +1,75 @@
+.card {
+  --animation: 0.2s ease;
+  --mask-opacity: 0.75;
+  --mask-opacity-hover: 0.6;
+  list-style-type: none;
+  display: block;
+  padding: 2em 1em;
+  margin: 1em 0;
+  border-radius: 12px;
+  background-position: center;
+  background-color: var(--bg2);
+  background-size: cover;
+  position: relative;
+  text-decoration: none;
+  text-shadow: 0 0 12px var(--bg0_h);
+}
+
+.card .title,
+.card .date {
+  color: var(--orange2);
+  position: relative;
+  z-index: 10;
+  transition: color var(--animation);
+  margin: 0;
+}
+
+.card .tag-list li {
+  transition: all var(--animation);
```

Comment:
should this just go in the tag list? Nah, I only want to animate tag-lists inside of cards. So this looks good.